test/Scrumptious.Testing.Data/SprintTest.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Scrumptious.Data.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Scrumptious.Testing.Data
+{
+    public class SprintTest
+    {
+
+        private Sprint sut;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make 'sut' 'readonly'. ([csharpsquid:S2933](https://sonarcloud.io/organizations/mrodsky-github/rules#rule_key=csharpsquid%3AS2933))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rodskyproj2&pullRequest=5&issues=AWVY2fiX2xJdnw6QkFKd&open=AWVY2fiX2xJdnw6QkFKd)

[ISSUE_KEY]: AWVY2fiX2xJdnw6QkFKd